scripts/generate_dump

```diff
@@ -189,25 +202,29 @@ save_cmd() {
         tarpath="${tarpath}.gz"
         filepath="${filepath}.gz"
         # cleanup_method will run in a sub-shell, need declare it first
-        local cleanup_method_declration=$(declare -f $cleanup_method)
         local cmds="$cleanup_method_declration; $cmd $redirect_eval | $cleanup_method | gzip -c > '${filepath}'"
         if $NOOP; then
             echo "${timeout_cmd} bash -c \"${cmds}\""
         else
             RC=0
             eval "${timeout_cmd} bash -c \"${cmds}\"" || RC=$?
-            if [ $RC -ne 0 ]; then
+            if [ $RC -eq 124 ]; then
```

Comment:
what is 124 stands for? is it known error code? if so why not using a constant here?

Reply:
Made a constant for it